inventory

```diff
@@ -66,7 +56,6 @@ ansible_service_broker_etcd_image_tag=latest
 
 #
 # If you run openshift deployment
-# You can add your master as schedulable node with option openshift_schedulable=true
-# Add at least one node with lable to run on it router and docker containers
-# openshift_node_labels="{'region': 'infra','zone': 'default'}"
+# Add at least one node with infra annotation to run on it router and docker containers
```

Comment:
the order should be: verb (run) object (router ...) place (on ...), i.e
`to run router and docker containers on it.`
or simply:
`to run router and docker containers there.`

Reply:
addressed